examples/simple/src/users/UserCreate.tsx

```diff
@@ -24,23 +24,18 @@ const UserEditToolbar = ({ permissions, ...props }) => {
         <Toolbar {...props}>
             <SaveButton
                 label="user.action.save_and_show"
-                mutationOptions={{
-                    onSuccess: data => {
-                        notify('ra.notification.created', 'info', {
-                            smart_count: 1,
-                        });
-                        redirect('show', data.id, data);
-                    },
-                }}
                 submitOnEnter={true}
             />
             {permissions === 'admin' && (
                 <SaveButton
                     label="user.action.save_and_add"
                     mutationOptions={{
                         onSuccess: data => {
-                            notify('ra.notification.created', 'info', {
-                                smart_count: 1,
+                            notify('ra.notification.created', {
+                                type: 'info',
+                                messageArgs: {
+                                    smart_count: 1,
+                                },
                             });
                             redirect(false);
```

Comment:
I think this redirect(false) isn't required anymore. But you need to add the form reset effect.